README.md

```diff
@@ -8,11 +8,13 @@ SantosPlatform was developed and tested on Ubuntu 14.04 LTS.  The current list o
 
 1. Install [SantosGUI](https://github.com/santosfamilyfoundation/SantosGUIs) onto your local machine.  Instructions for installing the front end is located in the [SantosGUI repository](https://github.com/santosfamilyfoundation/SantosGUI).
 2. Download Vagrant from its [downloads page](https://www.vagrantup.com/downloads.html).
-3. Clone this repo, [SantosInstallation](https://github.com/santosfamilyfoundation/SantosInstallation), which contains configuration files for Vagrant (auth.py, install.sh, Vagrantfile.LOCAL, Vagrantfile.AWS).
+3. Clone this repo, [SantosInstallation](https://github.com/santosfamilyfoundation/SantosInstallation), which contains configuration files for Vagrant (Vagrantfile and install.sh).
 4. Decide if your SantosPlatform will operate on a local machine, or on remote servers.
 
 ### 4a: Local installation (for testing and development)
-First, rename `Vagrantfile.LOCAL` to `Vagrantfile`. In the directory where SantosInstallation was cloned, run `vagrant up` to begin the install process. This step takes about 10 minutes to complete.
+First, download the Santos Vagrant box from [this link](https://goo.gl/6hl76J). Add it to your vagrant boxes with the command `vagrant box add santosbox santos.box`.
+
+In the directory where SantosInstallation was cloned, run `vagrant up` to begin the install process. This step takes about 10 minutes to complete.
```

Comment:
This instruction seems contradictory to the binary vagrant box.  We don't have to vagrant up if we already have a binary right?